src/libcommonserver/io/iohelper_win.cpp

```diff
@@ -583,7 +585,7 @@ static bool getRightsWindowsApi(const SyncPath &path, bool &read, bool &write, b
 
     // Get rights for trustee
     ACCESS_MASK rights = 0;
-    result = GetEffectiveRightsFromAcl(pfileACL, &IoHelper::getTrustee(), &rights);
+    //result = GetEffectiveRightsFromAcl(pfileACL, &IoHelper::getTrustee(), &rights);
```

Comment:
Why is it commented now?

Reply:
Fixed here: [4ad40c1](https://github.com/Infomaniak/desktop-kDrive/pull/381/commits/4ad40c1a17386c7afbcab654548d81295a6aa1ca)